apprentice_teacher.rb

```diff
@@ -1,6 +1,10 @@
-class ApprenticeTeacher
+require_relative 'all_teachers'
+require_relative 'feedback_module'
+
+class ApprenticeTeacher < All_Teachers
   attr_reader :age, :salary, :phase, :target_raise
   attr_accessor :name
```

Comment:
You can refactor the attr's, too!  Remember, they are just methods like anything else.